camera:
    fx: 609.799499511719
    fy: 609.5458984375
    dims: [480, 640]
segmentation:
    dia: .08
    radius_min_size: .01
    lower_bound: [200, 60, 110]
    upper_bound: [256, 100, 170]
subscribers:
    images: "~images"
publishers:
    pose: "~pose"